_plugins/fetch_docs.rb

```diff
@@ -108,11 +108,11 @@ def gen_versions_xml_file(filePath, sourceData)
 end
 
 forceDocumentationDownload = (ENV.fetch("FORCE_DOCUMENTATION_DOWNLOAD") { "true" }).upcase
-if forceDocumentationDownload == "FALSE" and File.exists?("docs/versions.xml")
```

Comment:
exists method was deprecated and removed in the latests versions of ruby. use file and directory instead